src/components/Notification/Notification.test.tsx

```diff
@@ -77,8 +77,12 @@ describe("Notification", () => {
   });
 
   it("renders with an Icon", () => {
-    // The Icon's role is "img".
-    expect(screen.queryByRole("img")).toBeInTheDocument();
```

Comment:
Is this no longer valid because the icon is now considered decorative?

Reply:
Yes, correct.